sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -222,6 +222,8 @@ class JDBCOptions(
 
   // User specified JDBC connection provider name
   val connectionProviderName = parameters.get(JDBC_CONNECTION_PROVIDER)
+
+  val prepareQuery = parameters.get(JDBC_PREPARE_QUERY).map(_ + " ").getOrElse("")
```

Comment:
add a brief comment to explain what this is for?

Reply:
+1

Reply:
Fixed in https://github.com/apache/spark/pull/36440/commits/aca8fa236f2de5144eb68e970f3b00255433ab7b